docs/developer-docs/latest/setup-deployment-guides/configurations/optional/typescript.md

```diff
@@ -0,0 +1,41 @@
+---
+title: TypeScript configuration - Strapi Developer Docs
+description: Details for the configuration of TypeScript projects
+sidebarDepth: 3
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/configurations/optional/sso.html
+---
+
+# TypeScript project configuration
+
+TypeScript-enabled Strapi applications have a specific [project structure](/developer-docs/latest/setup-deployment-guides/file-structure.md) with the following dedicated folders and configuration files:
+
+| TypeScript-Specific directories and files | Purpose                                                                                                                                           | Location         |
+|-------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|------------------|
+| `./dist` directory                        | Adds the location for compiling the project JavaScript source code.                                                                               | application root |
+| `build` directory                         | Contains the compiled administration panel JavaScript source code.  The directory is created on the first `yarn build` or `npm run build` command | `./dist/`         |
```

Comment:
Maybe it's just me, but at first, I thought it meant that we had a build folder at the root of the project. Why did we split the location and the directory/file name? Or maybe why don't we put them side by side instead of across the table?

Reply:
I changed the columns so that location is 2nd and purpose is 3rd.